certified-connectors/AvePoint Cloud Governance/apiProperties.json

```diff
@@ -28,6 +58,87 @@
         }
       }
     },
+    "connectionParameterSets": {
+      "uiDefinition": {
+        "displayName": "Authentication Type",
+        "description": "Please make sure the user account used for authentication is a Cloud Governance application administrator."
+      },
+      "values": [
+        {
+          "name": "oauth",
+          "uiDefinition": {
+            "displayName": "Azure Active Directory",
+            "description": "Sign in with Azure Active Directory"
+          },
+          "parameters": {
+            "token": {
+              "type": "oauthSetting",
+              "uiDefinition": {
+                "displayName": "Username"
+              },
+              "oAuthSettings": {
+                "clientId": "1892da54-7f57-4b6a-b6b0-fdc7f44e04c9",
```

Comment:
@sear-lin-avepoint secret look alike values are not allowed. Please replace with a dummy value.

Reply:
thanks for you reminder,  already replaced.